### ISSUE TITLE
Pip installed version doesn't work

### ISSUE THREAD

**Issue by TheGroundZero** [TheGroundZero/openvasreporting#4]
If you install openvasreporting through pip (`pip3 install OpenVAS-Reporting`) and then attempt to use the tool (`openvasreporting -i <xml files> -o <output file>`) you get an error:

```
Traceback (most recent call last):
  File "/usr/local/bin/openvasreporting", line 6, in <module>
    from openvasreporting import main
ImportError: cannot import name 'main' from 'openvasreporting' (/usr/local/lib/python3.7/dist-packages/openvasreporting/__init__.py)
```

Tested on Kali Linux and Ubuntu 

```
$ python3 --version
Python 3.7.4
$ pip3 --version
pip 19.2.1 from /usr/local/lib/python3.7/dist-packages/pip (python 3.7)
$ which openvasreporting 
/usr/local/bin/openvasreporting
```

Current resolution is to "run from source" by downloading the repo and running from there.

```
git clone https://github.com/TheGroundZero/openvasreporting.git
cd openvasreporting
python3 -m openvasreporting -i <xml files> -o <output file>
```

Preferred solution is one in which both methods (from source, using pip) work.

**Comment by SecT0uch** [comment]
Same here

**Comment by kk0nrad** [comment]
it seems that pip has version 1.4.3, not the latest

**Comment by TheGroundZero** [comment]
Should be fixed in 1.5.x
